src/latex2sympy/lib/__init__.py

```diff
@@ -1,11 +1,11 @@
 import platform
 
-if platform.system() == 'Linux':  # pragma: no cover
-    from latex2sympy.lib.linux.latex2antlrJson import parseToJson, LATEXLexerToken
-elif platform.system() == 'Darwin':  # pragma: no cover
-    if platform.machine() == 'arm64':  # pragma: no cover
-        from latex2sympy.lib.macOS.arm64.latex2antlrJson import parseToJson, LATEXLexerToken
-    else:
-        from latex2sympy.lib.macOS.x86_64.latex2antlrJson import parseToJson, LATEXLexerToken
+if platform.system() == "Linux":  # pragma: no cover
+    from latex2sympy.lib.linux.latex2antlrJson import LATEXLexerToken, parseToJson
+elif platform.system() == "Darwin":  # pragma: no cover
+    from latex2sympy.lib.macOS.latex2antlrJson import (  # noqa: F401
```

Comment:
@builtbywill is this a universal binary now?

Reply:
@jkdufair no, I just removed the x64 one since everyone is using arm64 macs now, just don't need it anymore